src/mime/mod.rs

```diff
@@ -139,6 +139,12 @@ impl FromStr for Mime {
     }
 }
 
+impl<'a> From<&'a str> for Mime {
+    fn from(value: &'a str) -> Self {
+        Self::from_str(value).unwrap()
```

Comment:
Could we use `expect` here instead? Probably copy over the message from `FromStr`'s error.